tests/phpunit/tests/cache.php

```diff
@@ -491,4 +491,33 @@ public function test_wp_cache_delete_multiple() {
 
 		$this->assertSame( $expected, $found );
 	}
+
+	/**
+	 * @ticket https://core.trac.wordpress.org/ticket/55999
+	 *
+	 * @covers wp_suspend_cache_addition
```

Comment:
```suggestion
	 * @covers ::wp_suspend_cache_addition
```

Though I'd say the coverage is more for `wp_cache_add` et al